medicalInstitution.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Medical Institution</h1>
+    <script>
+
+        class Person {
+            name;
+            surname;
+
+            constructor(name, surname) {
+                this.name = name;
+                this.surname = surname;
+            }
+        }
+
+        class Doctor extends Person {
+            specialization;
+
+            constructor(name, surname, specialization) {
+                super(name, surname);
+                this.specialization = specialization;
+                
+            }
+
+            makeAnAppointment(examination) {
+                console.log(`Appointment made for ${examination.type} for patient ${examination.patient} by doctor ${this.name}`)
+            }
+        }
+
+        class Patient extends Person {
+            jmbg;
+            numberOfHealthCard;
+            doctor;
+
+            constructor(name, surname, jmbg, numberOfHealthCard, doctor) {
+                super(name,surname);
+                this.jmbg = jmbg;
+                this.numberOfHealthCard = numberOfHealthCard;
+                this.doctor = doctor;
+            }
+
+            chooseDoctor(doctor1) {
+                this.doctor = doctor1;
+            }
+
+        }
+
+        class Examination {
+            date;
+            time;
+            patient;
+            type;
+
+            constructor(date, time, patient, type)
+            {
```

Comment:
Follow a convention of brackets, opening bracket should be in the same line as constructor